src/ess/imaging/io.py

```diff
@@ -0,0 +1,265 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright (c) 2024 Scipp contributors (https://github.com/scipp)
+
+from collections.abc import Callable, Iterable
+from enum import Enum
+from itertools import pairwise
+from pathlib import Path
+from typing import NewType
+
+import jax.numpy as np
```

Comment:
Missing in project dependencies, or is this unintentional?

Reply:
Yes it was unintentional...